traffic_portal/app/src/common/modules/widget/dashboardStats/widget.dashboardStats.tpl.html

```diff
@@ -19,43 +19,43 @@
 
 <div class="col-md-4 col-sm-6 col-xs-12">
     <div class="col-md-6 col-sm-12 col-xs-12 tile_stats_count current">
-        <span class="count_top">Current Bandwidth</span>
-        <div class="count">{{totalBandwidth}}</div>
+        <label for="totalBandwidth" class="count_top">Current Bandwidth</label>
+        <output id="totalBandwidth" class="count">{{totalBandwidth}}</output>
     </div>
     <div class="col-md-6 col-sm-12 col-xs-12 tile_stats_count current">
-        <span class="count_top">Current Connections</span>
-        <div class="count">{{totalConnections}}</div>
+        <label for="totalConnections" class="count_top">Current Connections</label>
+        <output id="totalConnections" class="count">{{totalConnections}}</output>
     </div>
 </div>
 <div class="col-md-8 col-sm-6 col-xs-12">
     <div class="col-md-4 col-sm-12 col-xs-12 tile_stats_count">
         <div class="col-md-6 col-sm-6 col-xs-6">
-            <span class="count_top"><a ng-click="navigateToPath('/cache-stats')">Healthy Caches</a></span>
-            <div class="count green">{{totalOnline}}</div>
+            <label for="totalOnline" class="count_top"><a ng-click="navigateToPath('/cache-stats')">Healthy Caches</a></label>
+            <output id="totalOnline" class="count green">{{totalOnline}}</output>
         </div>
         <div class="col-md-6 col-sm-6 col-xs-6">
-            <span class="count_top"><a ng-click="navigateToPath('/cache-stats')">Unhealthy Caches</a></span>
-            <div class="count red">{{totalOffline}}</div>
+            <label for="totaloffline" class="count_top"><a ng-click="navigateToPath('/cache-stats')">Unhealthy Caches</a></label>
+            <output id="totalOffline" class="count red">{{totalOffline}}</output>
         </div>
     </div>
     <div class="col-md-4 col-sm-12 col-xs-12 tile_stats_count">
         <div class="col-md-6 col-sm-6 col-xs-6">
-            <span class="count_top">Online Caches</span>
-            <div class="count">{{online()}}</div>
+            <label for="online-caches" class="count_top">Online Caches</label>
+            <output id="online-caches" class="count">{{online()}}</output>
```

Comment:
oh, here's your problem. should say {{online}} and not {{online()}}

Reply:
but it said `{{online()}}` before? If I do `{{online}}` it'll say something like `function online()` in the output.

Reply:
oh, you're right. sorry. i guess i was making assumptions why it wasn't working for me.